searchPage/search.html

```diff
@@ -1,24 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="search.css">
-        <script src="https://kit.fontawesome.com/2f327f6fc8.js" crossorigin="anonymous"></script>
-        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400&display=swap" rel="stylesheet">
-        <title>Document</title>
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="search.css" />
+    <script
+      src="https://kit.fontawesome.com/2f327f6fc8.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400&display=swap"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
     <header>
-        <nav class="header">
-            <span id="logo"><b>UNIGUIDE</b></span>
-            <i class="fas fa-align-right fa-flip-vertical menuIcon"></i>
+      <nav class="header">
+        <div>
+          <span id="logo"><b>UNIGUIDE</b></span>
+        </div>
 
-
-        </nav>
+        <div>
+          <i id="menuIcon" class="fas fa-align-right fa-flip-vertical"></i>
+        </div>
+      </nav>
     </header>
-    <script src="search.js"></script>
 
-</body>
-</html>
\ No newline at end of file
+    <div id="serach-container">
+      <div id="ima">
+        <img id="image" src="../img/Group 13.png" alt="" />
+      </div>
+
+      <div id="input-field">
+        <i id="search-icon" class="fas fa-search icon"></i>
+        <input
+          list="countries"
+          id="input"
+          name=""
+          placeholder="Enter country name"
+        />
+
+        <datalist id="countries">
```

Comment:
the list appears when the user clicks on the input field which I think should change, specially it appears with dark color and above countries pictures..
another suggestion: you can make a 'list sign' appear as a default and clicking on it lead to see the list